dashbuilder-webapp/src/main/java/org/dashbuilder/backend/SalesDataSetGenerator.java

```diff
@@ -134,7 +134,7 @@ private Date buildDate(int month, int year) {
         return c.getTime();
     }
 
-    private Date addDates(Date d, int days) {
+    private static Date addDates(Date d, int days) {
```

Comment:
Following sonar rule  squid:S2325 methods that don't access instance data should be static. But it is optional, I can remove this change from this commit if you would like.